src/ol/control/OverviewMap.js

```diff
@@ -45,8 +45,7 @@ const MIN_RATIO = 0.1;
  * @property {string|HTMLElement} [label='»'] Text label to use for the collapsed
  * overviewmap button. Instead of text, also an element (e.g. a `span` element) can be used.
  * @property {Array<import("../layer/Layer.js").default>|import("../Collection.js").default<import("../layer/Layer.js").default>} [layers]
```

Comment:
```suggestion
  * @property {Array<import("../layer/Layer.js").default>|import("../Collection.js").default<import("../layer/Layer.js").default>} layers
```

To mark the parameter as mandatory